Update styles.css

Grawl opened the pull request:
Add `word-break: break-all;` to `a` tag to avoid of page overflow by very long links that causes mobile view zoomed.

iamstarkov commented on styles.css:
```diff
@@ -240,3 +240,7 @@ tr {
+
+a {
+  word-break: break-all;
+}
```
can you define selector a bit more stricter? like `.container article a {}`

Grawl replied:
I think an `<article>` needs a classname such as `.post` or `.archive`. Don't like to style elements directly by DOM names.

iamstarkov replied:
i fully understand you, I will fix styling tags directly in few days. but for now is the best choice, at least proper amount of links will be affected. right now affect scope is too broad

Grawl replied:
Okay.

iamstarkov commented:
and please squash your commits into one

Grawl commented:
Edited from GitHub directly, sorry.

iamstarkov commented:
> Edited from GitHub directly, sorry.

its okay, will you squash?

Grawl commented:
Never before. `git rebase -i HEAD~3`?

iamstarkov commented:
git reset --mixed @~3
git add .
git commit -m ""

Grawl commented:
And submit as a new PR?

iamstarkov commented:
just push force to the same branch

Grawl commented:
Force push? I will survive it? :laughing:

iamstarkov commented:
you will

Grawl committed:
Update styles.css

Add `word-break: break-all;` to `a` tag to avoid of page overflow by very long links that causes mobile view zoomed.

Grawl commented:
Done.

iamstarkov commented:
thank you

Grawl commented:
Glad to help.
